jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/processinstance/JPAProcessInstanceManager.java

```diff
@@ -123,7 +123,8 @@ public ProcessInstance getProcessInstance(long id, boolean readOnly) {
                 ProcessInstanceInfo processInstanceInfo = (ProcessInstanceInfo) context.findProcessInstanceInfo( id );
                 if ( processInstanceInfo == null ) {
                     return null;
-                }                
+                }  
+                processInstance.setStartDate(processInstanceInfo.getStartDate());
```

Comment:
I would rather set this start date in org.jbpm.persistence.processinstance.ProcessInstanceInfo.getProcessInstance(InternalKnowledgeRuntime, Environment, boolean) ... and I see you dod that in the next file so why this is needed?